RetailCoder.VBE/UI/Command/MenuItems/ParserStateCommandBar.cs

```diff
@@ -47,16 +47,18 @@ private void OnRefresh()
 
         public void Initialize()
         {
-            var commandbar = _vbe.CommandBars.Add("Parsing", MsoBarPosition.msoBarTop, false, true);
+            var commandbar = _vbe.CommandBars.Add("Rubberduck", MsoBarPosition.msoBarTop, false, true);
```

Comment:
I'm completely ok with this, but then `ParserStateCommandBar` should be renamed `RubberduckCommandBar`.